### PR TITLE
Update Deutsch

### PR DESCRIPTION
**Pull request by marter001**
This translation is based on the default file of 

Version: 5.8-3156-gc113cffdf
Branch: dev
Commit: c113cffdf
Commit date: 2022-09-29

### CODE REVIEW

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -1373,9 +1370,9 @@ HISTORY_MSG_1061;(Lokal - CIECAM)\nSzenebasierte Bedingungen\nAbsolute Luminanz
 HISTORY_MSG_1062;(Lokal - CIECAM)\nSzenebasierte Bedingungen\nUmgebung
 HISTORY_MSG_1063;(Lokal - CIECAM)\nCAM16 - Farbe\nSättigung
 HISTORY_MSG_1064;(Lokal - CIECAM)\nCAM16 - Farbe\nChroma
-HISTORY_MSG_1065;(Lokal - CIECAM)\nCAM16 - Beleuchtung\nHelligkeit (J)
-HISTORY_MSG_1066;(Lokal - CIECAM)\nCAM16 - Beleuchtung\nHelligkeit (Q)
-HISTORY_MSG_1067;(Lokal - CIECAM)\nCAM16 - Kontrast\nKontrast (J)
+HISTORY_MSG_1065;(Lokal - CIECAM)\nHelligkeit J
+HISTORY_MSG_1066;(Lokal - CIECAM)\nHelligkeit
```
Comment (Floessie) [inline rtdata/languages/Deutsch:1374]: Was `Q` deliberately left out here?

File: rtdata/languages/Deutsch [hunk 2]
```diff
@@ -1571,7 +1568,7 @@ HISTORY_MSG_TRANS_METHOD;(Transformieren - Objektivkorrektur)\nMethode
 HISTORY_MSG_WAVBALCHROM;(Erweitert - Wavelet)\nRauschreduzierung\nFarb-Equalizer
 HISTORY_MSG_WAVBALLUM;(Erweitert - Wavelet)\nRauschreduzierung\nEqualizer Luminanz
 HISTORY_MSG_WAVBL;(Erweitert - Wavelet)\nUnschärfeebenen
-HISTORY_MSG_WAVCHR;(Erweitert - Wavelet)\nUnschärfeebenen\nChroma-Unschärfe
+HISTORY_MSG_WAVCHR;(Erweitert - Wavelet\nUnschärfeebenen\nChroma-Unschärfe
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Something went wrong here.

File: rtdata/languages/Deutsch [hunk 3]
```diff
@@ -2242,45 +2239,45 @@ TP_COLORAPP_ABSOLUTELUMINANCE;Leuchtdichte
...
+TP_COLORAPP_ALGO_JC;Helligkeit + Buntheit (JC)
 TP_COLORAPP_ALGO_JS;Helligkeit + Sättigung (JS)
 TP_COLORAPP_ALGO_QM;Helligkeit + Farbigkeit (QM)
 TP_COLORAPP_ALGO_TOOLTIP;Auswahl zwischen Parameter-Teilmengen\nund allen Parametern.
-TP_COLORAPP_BADPIXSL;Hot-/Dead-Pixelfilter
+TP_COLORAPP_BADPIXSL;Hot-/Bad-Pixelfilter
 TP_COLORAPP_BADPIXSL_TOOLTIP;Unterdrückt Hot-/Dead-Pixel (hell gefärbt).\n0 = Kein Effekt\n1 = Median\n2 = Gaussian\nAlternativ kann das Bild angepasst werden, um sehr dunkle Schatten zu vermeiden.\n\nDiese Artefakte sind auf Einschränkungen von CIECAM02 zurückzuführen.
 TP_COLORAPP_BRIGHT;Helligkeit (Q)
-TP_COLORAPP_BRIGHT_TOOLTIP;Helligkeit in CIECAM02/16 berücksichtigt die Weißintensität und unterscheidet sich von L*a*b* und RGB-Helligkeit.
+TP_COLORAPP_BRIGHT_TOOLTIP;Helligkeit in CIECAM berücksichtigt die Weißintensität.Sie unterscheidet sich von L*a*b* und RGB-Helligkeit.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: `.Sie` is missing a space.

File: rtdata/languages/Deutsch [hunk 4]
```diff
@@ -2242,45 +2239,45 @@ TP_COLORAPP_ABSOLUTELUMINANCE;Leuchtdichte
...
-TP_COLORAPP_DATACIE_TOOLTIP;Wenn aktiviert, zeigen die Histogramme der CIECAM02/16-Kurven die angenäherten Werte/Bereiche für <b>J oder Q</b> und <b>C, S oder M</b> NACH den CIECAM02/16-Anpassungen an. Das betrifft nicht das Haupt-Histogramm.\n\nWenn deaktiviert, zeigen die Histogramme der CIECAM02/16-Kurven die L*a*b*-Werte VOR den CIECAM02/16-Anpassungen an.
+TP_COLORAPP_CURVEEDITOR3_TOOLTIP;Korrigiert Buntheit, Sättigung oder Farbigkeit.\n\nZeigt das Histogramm der Chromatizität (L*a*b* ) VOR den CIECAM-Änderungen an.\nWenn 'CIECAM-Ausgabe-Histogramm in Kurven anzeigen' aktiviert ist, wird das Histogramm von C, S oder M NACH den CIECAM-Änderungen angezeigt.\n\nC, S und M werden nicht im Haupt-Histogramm angezeigt.\nFür die endgültige Ausgabe verwenden Sie das Haupt-Histogramm.
+TP_COLORAPP_DATACIE;CIECAM-Ausgabe-Histogramm in CAL-Kurven anzeigen
+TP_COLORAPP_DATACIE_TOOLTIP;Beeinflusst die Histogramme in FuB. Beeinflusst nicht das Haupthistogramm von RawTherapee.\n\nAktiviert: zeigt ungefähre Werte für J und C, S oder M NACH den CIECAM-Anpassungen.\nDeaktiviert: zeigt L*a*b*-Werte VOR den CIECAM-Anpassungen.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: `Haupthistogramm` (IMHO better) or `Haupt-Histogramm`?

File: rtdata/languages/Deutsch [hunk 5]
```diff
@@ -2373,7 +2370,7 @@ TP_COLORTONING_LUMA;Luminanz
 TP_COLORTONING_LUMAMODE;Luminanz schützen
 TP_COLORTONING_LUMAMODE_TOOLTIP;Wenn aktiviert, wird die Luminanz der Farben Rot, Grün, Cyan, Blau... geschützt.
 TP_COLORTONING_METHOD;Methode
-TP_COLORTONING_METHOD_TOOLTIP;L*a*b*-Überlagerung, RGB-Regler und RGB-Kurven\nverwenden eine interpolierte Farbüberlagerung.\n\nFarbausgleich (Schatten/Mitten/Lichter) und Sättigung\n(2-Farben) verwenden direkte Farben.
+TP_COLORTONING_METHOD_TOOLTIP;L*a*b*-Überlagerung, RGB-Regler und RGB-Kurven verwenden eine interpolierte Farbüberlagerung.\n\nFarbausgleich (Schatten/Mitten/Lichter) und Sättigung (2-Farben) verwenden direkte Farben.\n\Das Schwarz-Weiß-Werkzeug kann aktiviert werden, wenn eine beliebige Farbtönungsmethode verwendet wird, die eine Farbtönung ermöglicht.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: What is `\D`?

File: rtdata/languages/Deutsch [hunk 6]
```diff
@@ -2434,7 +2431,7 @@ TP_DIRPYRDENOISE_CHROMINANCE_MANUAL;Benutzerdefiniert
...
+TP_DIRPYRDENOISE_CHROMINANCE_METHOD_TOOLTIP;<b>Benutzerdefiniert:</b>\nManuelle Anpassung der Chrominanz-Rauschreduzierung.\n\n<b>Automatisch Global:</b>\nEs werden 9 Zonen für die Berechnung der Chrominanz-Rauschreduzierung verwendet.\n\n<b>Auto-Multizonen:</b>\nKeine Voransicht - wird erst beim Speichern angewendet. InAbhängig von der Bildgröße, wird das Bild in ca. 10 bis 70 Kacheln aufgeteilt. Für jede Kachel wird die Chrominanz-auschreduzierung individuell berechnet.\n\n<b>Vorschau:</b>\nNur der sichtbare Teil des Bildes wird für die Berechnung der Chrominanz-Rauschreduzierung verwendet.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Two new typos in this line.

File: rtdata/languages/Deutsch [hunk 7]
```diff
@@ -2742,7 +2739,7 @@ TP_LOCALLAB_AUTOGRAYCIE;Automatisch
 TP_LOCALLAB_AVOID;vermeide Farbverschiebungen
 TP_LOCALLAB_AVOIDCOLORSHIFT_TOOLTIP;Passt Farben an den Arbeitsfarbraum an und wendet die Munsell-Korrektur an (Uniform Perceptual Lab).\nMunsell-Korrektur ist deaktiviert wenn Jz oder CAM16 angewandt wird.
 TP_LOCALLAB_AVOIDMUN;Nur Munsell-Korrektur
-TP_LOCALLAB_AVOIDMUN_TOOLTIP;Munsell-Korrektur ist deaktiviert, wenn Jz or CAM16 angewandt wird
+TP_LOCALLAB_AVOIDMUN_TOOLTIP;Munsell-Korrektur ist deaktiviert, wenn Jz or CAM16 angewandt wird.
```
Comment (Floessie) [inline rtdata/languages/Deutsch]: Maybe `angewandt`/`angewendet` should be harmonized?

File: rtdata/languages/Deutsch [hunk 8]
```diff
@@ -3159,10 +3156,10 @@ TP_LOCALLAB_MASKLNOISELOW;In dunklen und hellen Bereichen verstärken
...
-TP_LOCALLAB_MASKLOWTHRESE_TOOLTIP;Dunklere Tonwertgrenze, unterhalb derer die Parameter nach und nach auf ihre ursprünglichen Werte wieder hergestellt werden, bevor sie durch die 'Dynamik und Belichtung'-Einstellungen geändert werden.\nSie können die Grauwerte mit verschiedenen Werkzeugen in ‘Maske und Anpassungen’ ändern: 'Glättradius', 'Gamma', 'Steigung‘ und 'Kontrastkurve'.\nVerwenden Sie einen 'feststellbaren Farbwähler' auf der Maske, um zu sehen, welche Bereiche betroffen sind. Stellen Sie sicher, dass Sie in den Einstellungen 'Hintergrundfarbmaske' = 0 festlegen.
+TP_LOCALLAB_MASKLOWTHRESE_TOOLTIP;Dunklere Tonwertgrenze, unterhalb derer die Parameter nach und nach auf ihre ursprünglichen Werte wieder hergestellt werden, bevor sie durch die 'Dynamik und Belichtung'-Einstellungen geändert werden.\nSie können die Grauwerte mit verschiedenen Werkzeugen in 'Maske und Anpassungen' ändern: 'Glättradius', 'Gamma', 'Steigung' und 'Kontrastkurve'.\nVerwenden Sie einen 'feststellbaren Farbwähler' auf der Maske, um zu sehen, welche Bereiche betroffen sind. Stellen Sie sicher, dass Sie in den Einstellungen 'Hintergrundfarbmaske' = 0 festlegen.
```
Comment (Floessie) [inline rtdata/languages/Deutsch:3159]: :+1:

**Review by Floessie — CHANGES_REQUESTED** [review]
Only nitpicking. Good job, @marter001!

File: rtdata/languages/Deutsch (hunk 1 above)
Comment (marter001) [reply rtdata/languages/Deutsch:1374]: yes, its not in default anymore, so i deleted it.

**Review by marter001 — COMMENTED** [review]
wow, quick and detailled compare. I revised the file and pls feel free for further comments.

File: rtdata/languages/Deutsch (hunk 7 above)
Comment (marter001) [reply rtdata/languages/Deutsch]: you're right, i switched them all to 'angewendet'

**Comment by marter001** [comment]
Sorry, i am not used to github yet. @Floessie is there anything else, i should do now?

**Comment by Beep6581** [comment]
GitHub tells me there is "1 review requesting changes", but that review simply says "good job", so I think there's nothing left for you to do @marter001 , we just wait for @Floessie to merge it.

**Comment by Floessie** [comment]
As mentioned elsewhere, I'm not at home ATM. Anyway, everything is fine, @marter001. Thanks for you work, that I'll merge any moment. :+1: 

Best,
Flössie

**Pull request merged**